src/Build/BackEnd/BuildManager/GraphBuildResult.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Build.Graph;
+
+namespace Microsoft.Build.Execution
+{
+    public sealed class GraphBuildResult
+    {
+        /// <summary>
+        /// Constructor creates a build result indicating a circular dependency was created.
```

Comment:
Woops. Should be "Constructor creates a build result with results for each graph node."